python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -587,6 +587,10 @@ def sum(self):
         """Compute the column-wise sum of the values in each group."""
         return self.agg("sum")
 
+    def prod(self):
+        """Compute the column-wise product of the values in each group."""
+        return self.agg("prod")
```

Comment:
Same question as in `aggregation.pyx`, is this necessary?

Reply:
This is necessary for pandas compatibility only.